index.html

```diff
@@ -0,0 +1,172 @@
+                  </li>
+                  <li class="nav-item">
+                      <a class="nav-link" href="https://web.archive.org/web/20210120125445/https://www.newsweek.com/newsgeek" title="">Newsgeek</a>
+                  </li>
+                  <li class="nav-item">
+                      <a class="nav-link" href="https://web.archive.org/web/20210120125445/https://www.newsweek.com/sports" title="">Sports</a>
+                  </li>
+                  <li class="nav-item">
+                      <a class="nav-link" href="https://web.archive.org/web/20210120125445/https://www.newsweek.com/health" title="">Health</a>
+                  </li>
+                  <li class="nav-item">
+                      <a class="nav-link" href="https://web.archive.org/web/20210120125445/https://www.newsweek.com/the-debate" title="">The Debate</a>
+                  </li>
+                  <li class="nav-item">
+                      <a class="nav-link" href="https://web.archive.org/web/20210120125445/https://www.newsweek.com/experts" title="">Experts</a>
+                  </li>
+                  <li class="nav-item">
+                      <a class="nav-link" href="https://web.archive.org/web/20210120125445/https://www.newsweek.com/vantage" title="">Vantage</a>
+                  </li>
+                </ul>
+                </div>
+            <div id="amp_suggestions_container1" class="marketplace-suggestion"></div>
+            <div class="trending-topics-wrapper">
+                <ul class="nav nav-tabs nav-justified">
+                    <li class="nav-item"><strong>Trending</strong></li>
+                    <li class="nav-item"><a class="nav-link" href="/web/20210120125445/https://www.newsweek.com/topic/joe-biden">Joe Biden</a></li>
+                    <li class="nav-item"><a class="nav-link" href="/web/20210120125445/https://www.newsweek.com/topic/donald-trump">Donald Trump</a></li>
+                    <li class="nav-item"><a class="nav-link" href="/web/20210120125445/https://www.newsweek.com/topic/inauguration">Inauguration</a></li>
+                    <li class="nav-item"><a class="nav-link" href="/web/20210120125445/https://www.newsweek.com/topic/coronavirus">Coronavirus</a></li>
+                    <li class="nav-item"><a class="nav-link" href="/web/20210120125445/https://www.newsweek.com/topic/stimulus-package">Stimulus Package</a></li>
+                    <li class="nav-item"><a class="nav-link" href="/web/20210120125445/https://www.newsweek.com/topic/vaccine">Vaccine</a></li>
+                    <li class="nav-item"><a class="nav-link" href="/web/20210120125445/https://www.newsweek.com/topic/capitol-riots">Capitol Riots</a></li>
+                </ul>
+            </div>
+            </nav>
+        </div>
+    </header>
+
+   </div>
+    
+    
+  
+  
+
+
+
+   <footer class="page-footer container-fluid">
```

Comment:
- [ ] Your footer should stick to the bottom of the page even if there is no content. You can make your footer stick to the bottom by adding the following bootstrap class `fixed-bottom`
```suggestion
    <footer class='page-footer container-fluid fixed-bottom'>
```